### PR TITLE
Fixes access to None object in validateEntity

### LINKED ISSUE #520: ERROR RetriesExhaustedError cause tool stop
Version: 2.2.1

We found that tool will stop if URI have some issue, and no html report created on this situation. 
Is it possible to catch this exception and let html report created anyway?

### PR DESCRIPTION
**Pull request by tomasg2012**
Fix #520

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## [2.2.2] - 2022-11-07
```
Comment (mraineri) [inline CHANGELOG.md:3]: Can you pull this out along with the version bump in RedfishServiceValidator.py? The build script takes care of this.

**Comment by mraineri** [comment]
@tomasg2012 what sort of conditions did this correct? Trying to understand what sort of behavior on the Redfish service shows the original issue?

**Comment by tomasg2012** [comment]
This doesn't change any major behavior except for getting the response status from an object that isn't None.  If a URI happened to be incorrect or the service failed at this step it wouldn't cause an exception.

**Comment by mraineri** [comment]
Got it, thanks.

Any thoughts on how to construct a set of mockups to simulate this?

**Comment by tomasg2012** [comment]
In any situation where a response would be None, and I think this would Only happen if the service experienced an exception.  This occurs in `traverse.rfService.callResourceURI`, and seems to have happened when retries were exhausted.  Not sure if the mockup can simulate this unless you terminated it manually during execution while validating any payload with links to Entities.

**Comment by jautor** [comment]
Approved 2/3

**Pull request merged**